zmq.c

```diff
@@ -207,7 +207,7 @@ php_zmq_context *php_zmq_context_get(zend_long io_threads, zend_bool is_persiste
 
 	if (is_persistent) {
 		zend_resource *le_p = NULL;
-		plist_key = strpprintf(0, "zmq_context=[%ld]", io_threads);
+		plist_key = strpprintf(0, "zmq_context=[%ld]", (long)io_threads);
 
```

Comment:
Proper fix to print zend_long is to use `ZEND_LONG_FMT`, but this macro is PHP 7 only